src/middlewares/index.js

```diff
@@ -0,0 +1,35 @@
+import { findById } from '../utils';
+import { User, Article } from '../models';
+
+/**
+ * @name verifyRecord
+ * @description This is a middleware for verifying if a recor. exists
+ * @param {object} req The request object
+ * @param {object} res The response object
+ * @param {function} next Passes the action to the next function
+ * @returns {object} Returns an error message
+ */
+const verifyRecord = async (req, res, next) => {
```

Comment:
We don't need this middleware since we can just go once and try to update. If the article doesn't exist, the database would throw an error that you can inspect and construct your response to the client.